doc/api/errors.md

```diff
@@ -409,12 +409,15 @@ The location information will be one of:
   its dependencies.
 
 The string representing the stack trace is lazily generated when the
-`error.stack` property is **accessed**.
+`error.stack` property is **accessed**.  
 
 The number of frames captured by the stack trace is bounded by the smaller of
 `Error.stackTraceLimit` or the number of available frames on the current event
 loop tick.
 
+However `error.stack` does not work on Microsoft,works only on  Firefox .
+
+
```

Comment:
I think that's off-topic – this is Node.js docs, we should not document of other runtimes behavior – and also wrong (`error.stack` is not defined by any standard, however every JS runtime implements it).

```suggestion
```

Reply:
Ok if that's the standard,but why studying  on w3schools it was specified under JavaScript errors ,I thought it was ok mentioning it.One time I tried logging it on my pc on a project tutorial I was following band it just never logged even though  it worked on the tutorial video.